frontend/src/features/ratings/RatingOrderByInput.tsx

```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+
+import {
+  FormControl,
+  InputLabel,
+  MenuItem,
+  Select,
+  SelectChangeEvent,
+} from '@mui/material';
+
+import { TitledSection } from 'src/components';
+import { useCurrentPoll } from 'src/hooks';
+import { getMetadataName } from 'src/utils/constants';
+
+interface FilterProps {
+  value: string;
+  onChange: (value: string) => void;
+}
+
+function RatingOrderByInput(props: FilterProps) {
+  const { t } = useTranslation();
+  const { name: pollName, options } = useCurrentPoll();
+
+  const extraMetadataOrderBy = options?.extraMetadataOrderBy ?? [];
+
+  const handleChange = (event: SelectChangeEvent) => {
+    props.onChange(event.target.value);
+  };
+
+  return (
+    <TitledSection title={t('ratingOrderByInput.orderBy')}>
+      <FormControl fullWidth size="small">
+        <InputLabel id="order-by-metadata-label">
+          {t('ratingOrderByInput.order')}
+        </InputLabel>
+        <Select
+          id="order-by-metadata"
+          labelId="order-by-metadata-label"
+          label={t('ratingOrderByInput.order')}
+          value={props.value}
+          onChange={handleChange}
+          inputProps={{ 'data-testid': 'order-by-metadata-input' }}
+        >
+          <MenuItem value="">--</MenuItem>
+          <MenuItem value="last_compared_at">
+            {t('ratingOrderByInput.lastComparisonDate')}&nbsp;
+            {t('ratingOrderByInput.(asc)')}
```

Comment:
I don't known if we want do that in this PR but I think a generic Select should be a good idea to simplify implementation. We should create a Select wich be implemented like this:

```
<GenericSelect 
    items=[{value: '', label: '--'}, {value: 'last_compared_at', label: <></>}]
/>
```

Example of simple implementation of this generic Select

```
export default function GenericSelect({items, array, label: string}){
  return <FormControl ...>
    <InputLabel>{labe}</InputLabel>
    <Select ...>
      {items.map(item => <MenuItem value={value}>{label}</MenuItem>)}
    </Select>
    </FormControl>
}
```

Reply:
As discussed with @Mayeul-T we won't create a generic component in this PR.